samples/model-builder/create_batch_prediction_job_sample.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence, Union
+
+from google.cloud import aiplatform
+
+#  [START aiplatform_sdk_create_batch_prediction_job_sample]
+def create_batch_prediction_job_sample(
+    project: str,
+    location: str,
+    model_resource_name: str,
+    job_display_name: str,
+    gcs_source: Union[str, Sequence[str]],
+    gcs_destination: str,
+    sync: bool = True,
+):
+    aiplatform.init(project=project, location=location)
+
+    batch_prediction_job = aiplatform.BatchPredictionJob.create(
+        model_name=model_resource_name,
+        job_display_name=job_display_name,
+        gcs_source=gcs_source,
+        gcs_destination_prefix=gcs_destination,
+        sync=sync,
```

Comment:
We should call `batch_prediction_job.wait()` if we allow the user to call this async. I'm not sure if it adds value here since we will have to wait to be able to print at least some of the information below.

Also, I don't have an opinion on which is a better option but I think we should try to keep usage consistent between samples. ie. add to the Dataset.create example above

Reply:
- Adding `wait()` call to this sample
- Adding `sync=sync` + `ds.wait()` to `create_and_import_dataset_image_sample`